build_external/scenarios/gaps_hi/master_guid

```diff
@@ -0,0 +1 @@
+00000000-0000-0000-0000-000000000000
```

Comment:
Please, add a blank line here.

Reply:
Why should the GUID files have newlines? (They work fine without it, is there some specific reason)

Reply:
The Github was informing the missing line, there is not a specific reason for this, probably the unique reason is to keep pattern with the other source code files.